### PR TITLE
Remove first api call on SDK initialisation

### PR DESCRIPTION
**Pull request by parvez301**
Remove first api call on initialisation

### CODE REVIEW

**Review by saumilhaptik — APPROVED** [review]

File: unleash-client-python/FeatureToggle/__init__.py [hunk 1]
```diff
@@ -339,14 +339,14 @@ def fetch_feature_toggles():
                             strategy_name = strategy.get('name', '')
                             parameters = strategy.get('parameters', {})
                             if strategy_name == 'EnableForPartners':
-                                partner_names = parameters.get('partner_names', '').replace(' ', '').split(',')
+                                partner_names = parameters.get('partner_names', '').replace(', ', ',').split(',')
```
Comment (sandesh89) [inline unleash-client-python/FeatureToggle/__init__.py:342]: Why not lstrip() or rstrip() here?

Comment (parvez301) [reply unleash-client-python/FeatureToggle/__init__.py:342]: lstrip or rstrip removes whitespaces from either beginning or ending.

I wanted to remove the whitespace from the middle of the string for eg `a b, c d` to `[a b, c d]`

Comment (sandesh89) [reply unleash-client-python/FeatureToggle/__init__.py:342]: Oh yeah since it returns an array it wont work

Comment (parvez301) [reply unleash-client-python/FeatureToggle/__init__.py:342]: yes

**Review by sandesh89 — APPROVED** [review]

**Pull request merged**